methods/array_ops/src/pg_gp/array_ops.sql_in

```diff
@@ -481,6 +481,18 @@ m4_ifdef(`__HAS_FUNCTION_PROPERTIES__', `NO SQL', `');
  * @brief This function takes an array as the input and executes element-wise multiplication by the scalar provided as the second argument, returning the resulting array. It requires that all the values are NON-NULL. Return type is the same as the input type.
  * @param x Array x
  * @param k Scalar
+ * @returns Array with each element of cos(x) multiplied by scalar.
```

Comment:
Thanks for pointing it out! This was however a 'draft' pull request.;-) And we already fix that in the latest version which have not been pushed yet. We also made quite a lot other changes as well.